module/item/systems/repair.js

```diff
@@ -274,25 +339,29 @@ class RepairData {
     }
 
     get repairDC() {
-        return this.diagram.system.craftingDC - repairModifier + this.enchantsDC
+        return this.diagram.system.craftingDC + repairModifier + this.enchantsDC
     }
 
-    get repairDCForumulae() {
-        let formulae = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${repairModifier}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
+    get repairDCForumula() {
+        let formula = `${this.diagram.system.craftingDC}[${game.i18n.localize('WITCHER.Repair.params.craftingDC')}] - ${Math.abs(repairModifier)}[${game.i18n.localize('WITCHER.Repair.params.repairMod')}]`
```

Comment:
instead of using Math.abs you can just add everything up

Reply:
But this is a string formula - putting '+' would interpolate into the '+' literal

Reply:
Ah I see. Then this is fine for me